Formula/r/runitor.rb

```diff
@@ -0,0 +1,18 @@
+class Runitor < Formula
+  desc "Command runner with healthchecks.io integration"
+  homepage "https://github.com/bdd/runitor"
+  url "https://github.com/bdd/runitor/archive/refs/tags/v1.3.0-build.4.tar.gz"
+  version "1.3.0-build.4"
+  sha256 "0dd2768ecfaeac14dda2e587d2ecc3cf28672e7ca86d117edb8633303144bc2b"
+  license "0BSD"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"runitor", "./cmd/runitor"
+  end
+
+  test do
+    system bin/"runitor", "-version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate teadd-gurkath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I've added a test for this, thanks